presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -593,13 +595,13 @@ public MaterializedResult execute(@Language("SQL") String sql)
     @Override
     public MaterializedResult execute(Session session, @Language("SQL") String sql)
     {
-        return executeWithPlan(session, sql, WarningCollector.NOOP).getMaterializedResult();
+        return executeWithPlan(session, sql, new DefaultWarningCollector(new WarningCollectorConfig())).getMaterializedResult();
```

Comment:
Why don't we want to use `NOOP` anymore?

Reply:
The new tests need to capture warnings.